manager/eris-mint/evm/abi/types.go

```diff
@@ -0,0 +1,27 @@
+package abi
```

Comment:
Should clarify in the notes below that this is an ABI for our SNative contracts.